kedro-org/kedro-viz
# Fix/multiple experiment tracking run menus

## Description

Resolves #894.

## Development notes

I've added a custom `useOutsideClick()` hook that will detect when a user clicks outside of a specific element, and when they do, a function will do something. In our case, it will close the menu that was open.

## QA notes

Open the Gitpod preview URL and ensure that you can't have multiple experiment tracking run menus open at the same time.

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/899"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
